code/computational_geometry/src/bresenham_line/Bresenham_Line_Algorithm.py

```diff
@@ -0,0 +1,72 @@
+# Import graphics library
+from graphics import *
```

Comment:
Try importing only the graphics functions that you're using in the code, instead of import *